docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -1,486 +1,488 @@
-# Gives Teleport the ability to create and delete certificationAuthority objects in the NTAuthCertificates container.
-dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CCDC;certificationAuthority;"
-# Gives Teleport the ability to write the cACertificate property in the NTAuthCertificates container.
-dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):WP;cACertificate;"
-
-```
-
-To save yourself time later, you can use the same prompt to get the security
-identifier of your new service account by running this command:
-
-```powershell
-Get-AdUser -Identity $SamAccountName | Select SID
-```
-
-Note this value (beginning with "S-") down, as it will be used as the `sid` field
-in the `ldap` section of your configuration file in a later step.
+   ```powershell
+   SID
+   ---
+   S-1-5-21-209875886-835680547-4265310078-1113
+   ```
+
+1. Copy the full security identifier—beginning with `S-`—returned. 
+   
+   You'll use this value for the `sid` field when you configure the `ldap` settings 
+   in a later step.
 
 ## Step 2/7. Prevent the service account from performing interactive logins
 
-<Admonition type="note" title="gpupdate.exe">
-  Throughout this step and the next one, you will be modifying GPOs, and
-  sometimes GPO modifications can take some time to propagate to all hosts. You
-  can force your changes to take effect immediately on your current host at any
-  time by opening a PowerShell prompt and running `gpupdate.exe /force` (though
-  their effects may still take time to propagate to other machines on the
-  domain).
-</Admonition>
+The next steps modify group policy objects (GPOs). Changes to group policies
+can take time to propagate to all hosts. You can force changes to take effect 
+immediately on your current host by opening PowerShell and running 
+`gpupdate.exe /force`. However, the change might still take time to propagate to other 
+hosts in the domain.
 
-The Teleport service account is only needed to authenticate over LDAP, meaning
-that it needn't be able to log in to Windows machines like an ordinary user.
-Restrict it from doing so by creating a new Group Policy Object (GPO) linked to
-your entire domain, and then deny it interactive login.
+The Teleport service account is only needed to authenticate over LDAP. The account
+doesn't need to log on to Windows computers like an ordinary user.
+You can prevent the service account from being used to log on by creating a new 
+Group Policy Object (GPO) linked to your entire domain, and then denying interactive 
+logins.
 
 ### Create a GPO
 
-1. Open a PowerShell prompt and change the `$GPOName` variable below to your
-   desired GPO name, or leave the recommended name:
+1. Open PowerShell and specify a name for the new group policy object with the `$GPOName` variable:
```

Comment:
Do you find it confusing that we have numbered steps inside of numbered steps (this is already step 2/7), or is this okay?

Reply:
Personally, I would do away with numbering the section headings with the "Step n/x." convention. I find it misleading because those "steps" are really "process" headings that contain steps and not steps themselves. But it's a major style change, so going for headings that include the fake Step broken down into step-by-step instructions seems less confusing (to me) than unnumbered paragraphs that resemble—but aren't identified as—steps.